pkg/kubelet/types/types.go

```diff
@@ -26,10 +26,9 @@ import (
 
 // TODO: Reconcile custom types in kubelet/types and this subpackage
 
-// HTTPGetter is an interface representing the ability to perform HTTP GET requests.
-type HTTPGetter interface {
-	// Get issues a GET to the specified URL.
-	Get(url string) (*http.Response, error)
+// HttpDoer encapsulates http.Do functionality
+type HttpDoer interface {
```

Comment:
If we do move forward with defining this interface, I believe go "best practice" is for the name to be stylized as `HTTPDoer`.